fastlane_bot/bot.py

```diff
@@ -964,14 +969,17 @@ def _handle_trade_instructions(
         # Get the flashloan token
         fl_token = calculated_trade_instructions[0].tknin_address
         fl_token_symbol = calculated_trade_instructions[0].tknin_symbol
+        flashloan_amount = int(calculated_trade_instructions[0].amtin_wei)
+        flashloan_fee = FLASHLOAN_FEE_MAP.get(self.ConfigObj.NETWORK, 0)
+        flashloan_fee_amt = int(flashloan_fee * flashloan_amount)
```

Comment:
In the initialization of `FLASHLOAN_FEE_MAP`, you are using a percentage value of 0.03% (at least by the comment next to it), i.e., a value between 0 and 100.
But the calculation in the line above implies that you are using a normalized value (between 0 and 1).
So either change the map to consist of normalized values, or add in this line a division by 100.